Nodejs/MigrateToJsps/Jsps/EsprojFileWriter.cs

```diff
@@ -110,9 +114,9 @@ private static JObject ReadPackageJson(string projectDir)
 
         private static string GetSdkVersion()
         {
-            // TODO: figure out if sdk is installed on machine?
-            // do i need to do that or will VS automatically install given version
-            throw new NotImplementedException();
+            // Use the installed version of JSPS on the NuGet fallback folder.
+            var versions = Directory.GetDirectories(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86), "Microsoft Visual Studio", "Shared", "NuGetPackages", "microsoft.visualstudio.javascript.sdk"));
+            return Path.GetFileName(versions.OrderByDescending(v => v).FirstOrDefault());
```

Comment:
I think this does lexicographic sorting so version 2 would be before version 10.